### PR TITLE
Add rowClass to table component

### PR DESCRIPTION
**Pull request by lab900-winand-vandenbergh**

### CODE REVIEW

File: projects/ui/src/lib/table/components/table/table.component.ts [hunk 1]
```diff
@@ -36,6 +38,9 @@ export class Lab900TableComponent implements OnChanges {
   @Input()
   public tableClass: string;
 
+  @Input()
+  public rowClass: propFunction<any> | string;
```
Comment (lab900-senn-lambrecht) [inline projects/ui/src/lib/table/components/table/table.component.ts:42]: Generic type ipv any

File: projects/ui/src/lib/table/components/table/table.component.ts [hunk 2]
```diff
@@ -12,6 +12,8 @@ import { Lab900TableCustomCellDirective } from '../../directives/table-custom-ce
 import { SortDirection } from '@angular/material/sort/sort-direction';
 import { Lab900TableTopContentDirective } from '../../directives/table-top-content.directive';
 
+type propFunction<T, R = string> = (data: T) => R;
```
Comment (lab900-senn-lambrecht) [inline projects/ui/src/lib/table/components/table/table.component.ts:15]: Isn't this already define somewhere else?

Comment (lab900-winand-vandenbergh) [reply projects/ui/src/lib/table/components/table/table.component.ts:15]: Yes, but it' s a different one, there it takes 2 arguments:
type propFunction<T, R = string> = (data: T, cell: TableCell) => R;

Should I define this in another way?

File: projects/ui/src/lib/table/components/table/table.component.ts (hunk 1 above)
Comment (lab900-winand-vandenbergh) [reply projects/ui/src/lib/table/components/table/table.component.ts:42]: There is no generic type in this component. Any is also used as type for the data here. E.g.:

  @Input()
  public data: any[];
    @Input()
  public selectedItems: any[];

**Review by lab900-senn-lambrecht — APPROVED** [review]

**Pull request merged**